frontend/src/components/SearchBar/FindMorePropertiesForm.tsx

```diff
@@ -0,0 +1,132 @@
+  margin-right: 10px;
+`;
+
+const FormSection = styled(props => <Container {...props} />)`
+  margin-top: 20px;
+  background-color: #e9ecef;
+  border-radius: 5px;
+`;
+
+const SearchButton = styled(props => <Button {...props} />)`
+  margin-top: 10px;
+  margin-left: 665px;
+`;
+
+/** This form is triggered by the FindMorePropertiesButton and contains additional filter fields for the PropertiesFilter */
+const FindMorePropertiesForm = <T extends any>(props: any) => {
+  const lookupCodes = useSelector<RootState, ILookupCode[]>(
+    state => (state.lookupCode as ILookupCodeState).lookupCodes,
+  );
+
+  const constructionType = _.filter(lookupCodes, (lookupCode: ILookupCode) => {
+    return lookupCode.type === API.CONSTRUCTION_CODE_SET_NAME;
+  }).map(mapLookupCode);
+  const predominateUses = _.filter(lookupCodes, (lookupCode: ILookupCode) => {
+    return lookupCode.type === API.PREDOMINATE_USE_CODE_SET_NAME;
+  }).map(mapLookupCode);
+
+  const { handleSubmit } = useFormikContext();
+
+  return (
+    <>
+      <p>
+        Search for properties within the Enhanced Referral Process (ERP) or Surplus Property Program
+        (SPP)
+      </p>
+      <FormSection>
+        <Row style={{ marginLeft: 10 }}>
+          <h6>Search by</h6>
+        </Row>
+        <StyledRow style={{ marginLeft: 35 }}>
+          <Input field="administrativeArea" label="Location" />
+          <VerticalLine />
+          <ProjectNumber field="projectNumber" label="Project number" placeholder="SPP #" />
+        </StyledRow>
+      </FormSection>
+      <FormSection>
+        <Row style={{ marginLeft: 10 }}>
+          <h6>Building search criteria</h6>
+        </Row>
+        <StyledRow>
+          <StyledSelect
+            field="predominateUseId"
+            placeholder="Any"
+            options={predominateUses}
+            label="Predominant use"
+          />
+          <VerticalLine />
+          <NumberInput label="Number of floors" field="floorCount" />
+        </StyledRow>
+        <StyledRow>
+          <StyledSelect
+            field="constructionTypeId"
+            placeholder="Any"
+            options={constructionType}
+            label="Construction type"
+          />
+          <VerticalLine />
+          <NumberInput label="Net rentable area" field="rentableArea" />
+        </StyledRow>
+      </FormSection>
+      <FormSection>
+        <Row style={{ marginLeft: 10 }}>
+          <h6>Land search criteria</h6>
+        </Row>
+        <StyledRow style={{ marginLeft: -13 }}>
+          <NumberInput label="Lot size" field="minLotSize" placeholder="min" />
+          <span style={{ marginTop: 5, marginLeft: 5, marginRight: 5 }}>-</span>
+          <NumberInput field="maxLotSize" placeholder="max" />
+          <VerticalLine />
+          <Check label="Bare land only" field="bareLandOnly" />
+        </StyledRow>
+      </FormSection>
+      <Row>
+        <SearchButton onClick={handleSubmit}>Search</SearchButton>
+      </Row>
```

Comment:
is there a way you could have the search form close on "Search" so that it wasn't blocking the map anymore?

Reply:
Yup this is possible, just wondering if they would get confused if it is closed right away though.. think I will leave as is for now and there can be a new UX ticket after feedback